waiter/resources/web/maintenance.html

```diff
@@ -0,0 +1,177 @@
+<!doctype html>
+<html>
+<head>
+    <title>"<%= token %>" in Maintenance Mode</title>
```

Comment:
Nitpick: avoid quotes in title.
We should include the service name on this page, e.g. additional info.